content/guides/api-calls/pagination/marker-based.md

```diff
@@ -95,8 +95,9 @@ results as an array, as well as some information about the current page of resul
 
 ## Example endpoints
 
-Some endpoints that support offset-based pagination are:
+Some endpoints that support marker-based pagination are:
 
 - [List items for a folder](endpoint://get_folders_id_items)
 - [List a file's collaborations](endpoint://get-files-id-collaborations)
 - [List all webhooks for a user](endpoint://get-webhooks)
+- [List all trashed items for a user](endpoints://trash)
```

Comment:
I assume you mean this endpoint here? https://developer.box.com/en/reference/get-folders-trash-items/ 

In this case, the url should be: `endpoint://get-folders-trash-items` (the slug of the url)

Could you update this? Then I'll merge the changes.